clustering/web/undertow/src/main/java/org/wildfly/clustering/web/undertow/session/DistributableSessionManagerFactory.java

```diff
@@ -49,6 +51,7 @@
  * @author Paul Ferraro
  */
 public class DistributableSessionManagerFactory implements io.undertow.servlet.api.SessionManagerFactory {
+    private static final String TOLERATE_SESSION_CREATION_AFTER_REDIRECT_PROPERTY = "jboss.web.tolerate-session-creation-after-redirect";
```

Comment:
I'm assuming this property is a temporary property so we can fix it for now until the config can be added to the model, am I right?

Reply:
Yes - I anticipate replacing this an Undertow management attribute that configures: https://github.com/pferraro/undertow/blob/master/servlet/src/main/java/io/undertow/servlet/api/DeploymentInfo.java#L1424